csv/result.go

```diff
@@ -31,6 +31,7 @@ const (
 	datatypeAnnotation = "datatype"
 	groupAnnotation    = "group"
 	defaultAnnotation  = "default"
+	errorAnnotation    = "error"
```

Comment:
The `error` is not an annotation. It will never appear in the first column of the csv data if annotations are being used.

Reply:
I'll rename this. It's not really an annotation as I use it.